plugins/dell-dock/dell-dock.quirk

```diff
@@ -115,3 +125,16 @@ DellDockUnlockTarget = 10
 DellDockHubVersionLowest = 1.31
 DellDockBlobMajorOffset = 0x400a
 DellDockBlobMinorOffset = 0x4009
+
+# USB4 controller
+[TBT-00d4b071]
+Name = USB4 controller in Dell dock
+Summary = USB4 controller
+Vendor = Dell Inc.
+VendorId = TBT:0x00D4
+ParentGuid = USB\VID_413C&PID_B06E&hub&embedded
+FirmwareSizeMin=0x40000
+FirmwareSizeMax=0x80000
+Flags = skips-restart,require-ac,dual-image
+Icon = thunderbolt
+InstallDuration = 46
```

Comment:
I thought the extended duration for `tbt` is a buffer therefore the `InstallDuration` here changed to `46` (previously `26` takes account of the `20` sec tolerance)

```
	/* Thunderbolt SKU takes a little longer */
	if (self->data->module_type == MODULE_TYPE_130_TBT ||
	    self->data->module_type == MODULE_TYPE_45_TBT  ||
	    self->data->module_type == MODULE_TYPE_130_USB4) {
		guint64 tmp = fu_device_get_install_duration (device);
		fu_device_set_install_duration (device, tmp + 20);
	}

```